proposals/accepted/047-standard-library-reform.rst

```diff
@@ -233,8 +233,8 @@ This is what we should emulate in order to provide a top-tier programming enviro
 Prior attempts at splitting ``base``
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-There have been prior attempts to split ``base`` before, but they attempted to get everything done at once, setting a dangerously high bar for success.
-This approach here, by contrast, first and foremost seeks to the difficulties and find a sustainable, suitably low risk approach.
+There have been attempts to split ``base`` before, but they attempted to get everything done at once, setting a dangerously high bar for success.
+This approach here, by contrast, first and foremost seeks to avoid the difficulties and find a sustainable, suitably low-risk approach.
```

Comment:
```suggestion
This approach here, by contrast, first and foremost seeks to avoid those difficulties and find a sustainable, suitably low-risk approach.
```